content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -325,6 +325,34 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### volume.kubernetes.io/selected-node
+
+Used on: PersistentVolumeClaim
+
+This annotation is added to a PVC that has been triggered by scheduler to be dynamically provisioned. Its value is the name of the selected node.
+
+### pv.kubernetes.io/bind-completed
+Example: `pv.kubernetes.io/bind-completed: "true"`
+
+Used on: PersistentVolumeClaim
+
+The value for this annotation must be true to take effect. This annotation indicates that {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}} was successfully bound to a PersistentVolume.
+
+### pv.kubernetes.io/bound-by-controller
+Example: `pv.kubernetes.io/bound-by-controller: "true"`
+
+Used on: PersistentVolume or PersistentVolumeClaim
+
+Value can either be `true` or `false`.
+This annotation applies to PVs and PVCs. It indicates that the binding (PV->PVC or PVC->PV) was installed by the controller. The absence of this annotation means the binding was done by the user (i.e. pre-bound). Value of this annotation does not matter. External PV binders must bind PV the same way as PV controller, otherwise PV controller may not handle it correctly.
+
+### pv.kubernetes.io/provisioned-by
+Example: `pv.kubernetes.io/provisioned-by: "flexVolume"`
+
+Used on: PersistentVolume
+
+This annotation is added to a PV that has been dynamically provisioned by Kubernetes. Its value is name of volume plugin that created the volume. It serves both user (to show where a PV comes from) and Kubernetes (to recognize dynamically provisioned PVs in its decisions).
```

Comment:
```suggestion
This annotation is added to a {{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}}
(PVs) that has been dynamically provisioned by Kubernetes. Its value is the name of volume plugin that
created the volume. It serves both users (to show where a PV has come from) and Kubernetes itself
(so that the control plane can identity PVs that were dynamically provisioned).
```